### ISSUE TITLE
[iceprod2] docs warning cleanup (Trac #1369)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1361]
Building the docs generates a lot of warnings, mostly due to docstring errors.  Fix them up and make the warnings go away.

Migrated from https://code.icecube.wisc.edu/ticket/1369
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:28", 
    "description": "Building the docs generates a lot of warnings, mostly due to docstring errors.  Fix them up and make the warnings go away.", 
    "reporter": "david.schultz", 
    "cc": "ddelventhal", 
    "resolution": "fixed", 
    "_ts": "1550067088921308", 
    "component": "iceprod", 
    "summary": "[iceprod2] docs warning cleanup", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2015-09-24T20:29:45", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "cleanup"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20151204T23:14:44: 
* david.schultz changed status from "new" to "closed"
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> In changeset 140119:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="140119"
> docs updates. this and lots of previous commits closes #1361
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:28: 
* olivas changed milestone from "IceProd 2.0" to ""
* olivas commented:

> Milestone IceProd 2.0 deleted
